### ISSUE TITLE
#-174373911------------- Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1342]
<strong>Source:</strong>`<span class="c-coordinateJobPanel__gridFieldText ng-star-inserted" title="Always New Execute GP Service">Always New Execute GP Service</span>`

<strong>Element Location: </strong>.c-coordinateJobPanel__gridFieldText[title="Always New Execute GP Service"]

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 4.28 (foreground color: #6a6a6a, background color: #c7eaff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,TTv5,TT13.c,EN-301-549,EN-9.1.4.3,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=axeAPI
